src/style/app.module.css

```diff
@@ -3772,6 +3858,21 @@ button[data-testid='createPostBtn'] {
   border: none;
 }
 
+.leftDrawer .optionList button:hover {
+  background-color: var(--grey-light) !important;
+  box-shadow: var(--hover-shadow);
+  color: black !important;
+}
+.leftDrawer .optionList button:active,
+.leftDrawer .optionList button:focus {
+  background-color: var(--grey-light) !important;
+  color: black !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`